pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -55,6 +57,11 @@ const (
 	requiredMCPCertCheckFreq = 500 * time.Millisecond
 )
 
+var (
+	useExternalGalley = env.RegisterBoolVar("PILOT_EXTERNAL_GALLEY", true,
+		"Set to restore the use of a standalone galley, if istio-galley.istio-system is the source.")
```

Comment:
move to features please. Otherwise, this will stay here and no one will know to enable this.

Reply:
It's only used for 1-2 weeks until dashboard tests are fixed. This is not intended to be enabled by users, just to ease the integration of istiod.

Reply:
Removed for now ( together with in process galley, to resolve this PR)